pulsar-client-cpp/wireshark/pulsarDissector.cc

```diff
@@ -445,10 +515,10 @@ static int dissect_pulsar_message(tvbuff_t *tvb, packet_info* pinfo, proto_tree*
         }
         case BaseCommand::SEND: {
             const CommandSend& send = command.send();
-            RequestData& data = state->producers[send.producer_id()].messages[send.sequence_id()];
-            data.requestFrame = pinfo->fd->num;
-            data.requestTimestamp.secs = pinfo->fd->abs_ts.secs;
-            data.requestTimestamp.nsecs = pinfo->fd->abs_ts.nsecs;
+            RequestData& request_data = state->producers[send.producer_id()].messages[send.sequence_id()];
+            request_data.requestFrame = pinfo->fd->num;
+            request_data.requestTimestamp.secs = pinfo->fd->abs_ts.secs;
+            request_data.requestTimestamp.nsecs = pinfo->fd->abs_ts.nsecs;
```

Comment:
There's no need to rename `data` to `request_data`.

Reply:
```c++
static int dissect_pulsar_message(tvbuff_t *tvb, packet_info* pinfo, proto_tree* tree, void* data _U_)
```
Here has a `data` too, it will make this can be confusing.

Reply:
The `data` variable is passed to

```c++
link_to_response_frame(cmd_tree, tvb, cmdOffset, cmdSize, data);
```

```c++
void link_to_response_frame(proto_tree* cmd_tree, tvbuff_t* tvb, int offset, int size,
                            const RequestData& reqData) {
```

I don't think there's something confusing.

Reply:
And if you want to keep the same coding style, you should have noticed in the other `case` blocks, the variable name `reqData` is used. However, as a local variable whose lifetime is just inside the `case` block, the name `data` is just okay.

Reply:
I change to `reqData ` now.